# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ArmActionGoal action_goal
ArmActionResult action_result
ArmActionFeedback action_feedback
